Makefile

```diff
@@ -310,7 +336,7 @@ install/lib/libredis++.a:
 	mkdir -p compile && \
 	cd compile && \
 	cmake -DCMAKE_BUILD_TYPE=Release -DREDIS_PLUS_PLUS_BUILD_TEST=OFF -DREDIS_PLUS_PLUS_BUILD_SHARED=OFF -DCMAKE_PREFIX_PATH="../../../install/lib/" -DCMAKE_INSTALL_PREFIX="../../../install" -DCMAKE_CXX_STANDARD=17 .. && \
```

Comment:
For consistency, build all the dependencies that are built using CMake using the `cmake --build` pattern used to build Smartredis

Reply:
I stuck to the existing build patterns we used previously to make sure the builds worked